db-cleanup/airflow-db-cleanup.py

```diff
@@ -46,7 +51,7 @@
 }
 
 dag = DAG(DAG_ID, default_args=default_args, schedule_interval=SCHEDULE_INTERVAL, start_date=START_DATE)
-
+dag.doc_md = __doc__
```

Comment:
Is \_doc\_ defined somewhere?

Reply:
Moved the comments to the top of the file and the documentation is now showing up in the Graph view page.

Reply:
Ok we'll need this to be done for the other dags too

Reply:
Updated remaining DAGs with the dag.doc_md changes.